src/lib.rs

```diff
@@ -78,7 +78,7 @@
 //! [`server::handshake`]: server/fn.handshake.html
 //! [`client::handshake`]: client/fn.handshake.html
 
-#![doc(html_root_url = "https://docs.rs/h2/0.3.10")]
+#![doc(html_root_url = "https://docs.rs/h2/0.3.11")]
```

Comment:
FWIW we could probably drop these directives per https://github.com/rust-lang/api-guidelines/pull/230